doc/telescope_changelog.txt

```diff
@@ -195,5 +195,64 @@ https://github.com/stevearc/dressing.nvim which has support for multiple
 different backends including telescope.
 
 
+                                                    *telescope.changelog-1945*
+
+Date: July 01, 2022
+PR: https://github.com/nvim-telescope/telescope.nvim/pull/1945
+
+This is our dev branch which contains a lot of PRs, a lot of fixes,
+refactoring and general quality of life improvements. It also contains new
+features, the most noteworthy are the following (mostly developed by the
+community):
+- feat: none strategy & control attachment (#1867)
+- feat: force buffer delete for terminal and improvements for
+  Picker:delete_selection (#1943)
+- feat(tags): process tagfiles on the fly (#1989)
+- feat(builtin.lsp): implement builtin handlers for
+  lsp.(incoming|outgoing)_calls (#1484)
+- feat: clear previewer if no item is selected (#2004)
+- feat: add min max boundary to width, height resolver (#2002)
+- feat: Add entry_index for entry_makers (#1850)
+- feat: refine with new_table (#1115)
+
+The last one is one of the most existing new features, because it  allows  you
+to go from live_grep into a fuzzy environment with the following mapping
+`<C-Space>`. It's a general interface we now implemented for `live_grep` and
+`lsp_dynamic_workspace_symbols` but it could also be easily implemented for
+other builtins, by us or the user. It's now available for extension developers.
+We will add documentation in the next couple of days and improve it by adding
+more options to configure it after the initial 0.1 release.
```

Comment:
I hope you will also add a name for it since the current anonymous function
1. looks weird in `which-key` 
2. cannot be remapped (which is a problem on macOS where `c-space` is a default system shortcut)

:D

Reply:
yeah makes sense. I'll make the proposed changes